### DISCUSSION TITLE
Vue hooks is not triggering

### DISCUSSION THREAD

**Discussion by sandyksit** [directus/directus#16598] [Archive]
Hi Team, @rijkvanzanten 
I am using version 9.18.1. I have interface extension and using vue life cycle 'updated' method, but it's not triggering when any collection field is update.

**Comment by br41nslug** [comment]
The `onUpdated` callback is not super helpful in the context of an interface as the data in an interface does not get updated unless by the interface itself or a page reload. 
> triggering when any collection field is update.

Also interfaces will only work for the one field they're linked to.
What is the goal with this interface you're building?
